app/api/models/domain/bfLoss/LossType.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package api.endpoints.bfLoss.domain.anyVersion
+package api.models.domain.bfLoss
```

Comment:
I think even if these are the same for v3 and v4, they ought to be in both the v3 and v4 package. Less chance that one of them would be accidentally modified for, say, a new v5 in the future.

(The v4 tests would likely break so it would be caught, but even so it kind of makes more sense, in this particular case, for them to be duplicated, as they're version-specific)